src/naming/utils.cairo

```diff
@@ -57,4 +58,10 @@ impl UtilsImpl of UtilsTrait {
         let next = self.get_chars_len(p);
         1 + next
     }
+
+    fn get_altcoin_price(
+        self: @Naming::ContractState, altcoin_quote: Wad, domain_price_eth: Wad
+    ) -> u256 {
+        (domain_price_eth / altcoin_quote).into()
```

Comment:
It should be a multiplication here. The base currency is ETH, the quoted currency is something like USDC. In that context a quote means the price of 1 ETH in the quoted currency, so for example 3500 USDC. It means you should just have to do a multiplication. If the API called by the backend returns the other way (the altcoin is the base currency), we should modify the backend to return 1/result, this will help keep this code less confusing. Also a multiplication should be significantly cheaper.